helm/hpcc/values.yaml

```diff
@@ -1,5 +1,9 @@
 # Default values for hpcc.
 
+#tags:
```

Comment:
This is a top level mechanism we could utilize to switch dependencies on/off.
This PR sets up a more verbose flag instead: global.logging.managedElasticStack.enabled